Example/PaymentSheet Example/PaymentSheet Example/ExampleCustomDeferredCheckoutViewController.swift

```diff
@@ -0,0 +1,334 @@
+//
+//  ExampleCustomDeferredCheckoutViewController.swift
+//  PaymentSheet Example
+//
+//  Created by Nick Porter on 3/27/23.
+//
+
+import Foundation
+//  Note: Do not import Stripe using `@_spi(STP)` in production.
+//  This exposes internal functionality which may cause unexpected behavior if used directly.
+// TODO(porter) Remove STP usage before GA
+@_spi (STP) import StripePaymentSheet
+import UIKit
+
+class ExampleCustomDeferredCheckoutViewController: UIViewController {
+    @IBOutlet weak var buyButton: UIButton!
+    @IBOutlet weak var paymentMethodButton: UIButton!
+    @IBOutlet weak var paymentMethodImage: UIImageView!
+
+    @IBOutlet weak var hotDogQuantityLabel: UILabel!
+    @IBOutlet weak var saladQuantityLabel: UILabel!
+    @IBOutlet weak var hotDogStepper: UIStepper!
+    @IBOutlet weak var saladStepper: UIStepper!
+    @IBOutlet weak var subtotalLabel: UILabel!
+    @IBOutlet weak var salesTaxLabel: UILabel!
+    @IBOutlet weak var totalLabel: UILabel!
+    @IBOutlet weak var subscribeSwitch: UISwitch!
+
+    private var paymentSheetFlowController: PaymentSheet.FlowController!
+    private let baseUrl = "https://stripe-mobile-payment-sheet-custom-deferred.glitch.me"
+    private var backendCheckoutUrl: URL {
+        return URL(string: baseUrl + "/checkout")!
+    }
+    private var createIntentUrl: URL {
+        return URL(string: baseUrl + "/create_intent")!
+    }
+    private var computeTotalsUrl: URL {
+        return URL(string: baseUrl + "/compute_totals")!
+    }
```

Comment:
nit: I think these should be `let`s

Reply:
Computed var's can't be let's.

Reply:
Right, but why do they need to be computed vars?  They shouldn't change.

Reply:
Can't be let's since the variables use `baseUrl` which is an instance variable. We have a few options.

1. Make `baseUrl` static
2. Define like `var computeTotalsUrl = URL(string: "https://stripe-mobile-payment-sheet-custom-deferred.glitch.me/compute_totals)`
3. Leave as-is

Reply:
1 sounds right to me.